src/components/Login.js

```diff
@@ -1,8 +1,127 @@
 import React from "react";
+import { useState, useContext } from "react";
 import "../App.css";
+import { FirebaseContext } from "./Context/Firebase";
+import UserContext from "./Context/UserContext";
+import TextField from "@material-ui/core/TextField";
+import Checkbox from "@material-ui/core/Checkbox";
+import Button from "@material-ui/core/Button";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+
+import { Redirect } from "react-router-dom";
 
 function Login() {
-  return <div className="login">Login page</div>;
+  const firebase = useContext(FirebaseContext);
+  const userData = useContext(UserContext);
+  const [newUser, createUser] = useState(false);
+  const toggle = () => createUser(!newUser);
+  const [signinInputs, setSignin] = useState({
+    email: "",
+    password: "",
+  });
+
+  const handleChangeIn = name => event => {
+    setSignin({ ...signinInputs, [name]: event.target.value });
+  };
+
+  function signin(email, pass) {
+    firebase.doSignInWithEmailAndPassword(email, pass).catch(error => {
+      console.log(error);
+    });
+  }
+
+  function signup(email, pass) {
+    firebase
+      .doCreateUserWithEmailAndPassword(email, pass)
+      .then(authUser => {
+        console.log(userData);
```

Comment:
console.log() statements should be deleted

Reply:
This is important to leave in until its published to master as its needed for verifying database access and flow